packages/web/src/pages/MessageDetail/index.tsx

```diff
@@ -50,17 +50,17 @@ const MessageForm: React.FC<Props> = ({ match }) => {
       ...data,
       sentAtFormatted: data.sentAt
         ? format(parseISO(data.sentAt), "dd/MM/yyyy HH:mm'h'")
-        : 'Não enviada'
+        : 'Não enviada',
     };
   }, [data]);
 
   const handleSendMessage = useCallback(async () => {
     try {
       const response = await axios.post(`/messages/${id}/send`);
 
-      mutate(requestKey, response, false); 
+      mutate(requestKey, response, false);
 
-      console.log('mutate called')
+      console.log('mutate called');
```

Comment:
I think we can remove this `console.log` and any others that you find in the code.

Reply:
Ok, I will remove all `console.log`